libraries/AP_Notify/OreoLED_I2C.h

```diff
@@ -19,6 +19,10 @@
 #include <AP_HAL/AP_HAL.h>
 #include "NotifyDevice.h"
 
+#ifndef HAL_WITH_OREO_LED
+#define HAL_WITH_OREO_LED !HAL_MINIMIZE_FEATURES
```

Comment:
add a flash size 1024 check for that too I think

Reply:
I actually just turned it off as per above. I put the flash check in for display though